docs/how-to-guides/isis-developer-guides/writing-isis-tests-with-ctest-and-gtest.md

```diff
@@ -40,7 +40,7 @@ For the rest of this document, we will use `appname` as the name of the applicat
 1. In `appname.h` and `appname.cpp` create a new function in the `Isis` namespace with the following signature `void appname(UserInterface &ui)`. If the application has a `from` cube, add a second function with the input cube as the first argument `void appname(Cube incube, UserInterface &ui)`. 
 1. Copy the contents of the existing `IsisMain` function currently located in `main.cpp` into the new `void appname(Cube incube, UserInterface &ui)` function. (if the app doesn't take any file parameters (i.e., network, text, cube list...) copy the contents into` void appname(UserInterface &ui)`). If there is no input cube, but there are other input files, add them as input parameters similar to how the input cube was done, see spiceinit and cnetcheck tests for examples.  
 1. Modify `void appname(UserInterface &ui)` to open the input cube, usually "from", and/or other files and call the second function `void appname(Cube incube, UserInterface &ui)` 
-1. Copy any helper functions or global variables from `main.cpp` into `appname.cpp`. This is to not pollute the `Isis` namespace and avoid redefining symbols
+1. Copy any helper functions or global variables from `main.cpp` into `appname.cpp`. Anything that was a global variable should be declared as static, this scopes them to the `.cpp` file. This is to not pollute the `Isis` namespace and avoid redefining symbols. 
```

Comment:
Can you also change "not pollute" to "avoid polluting"
```suggestion
1. Copy any helper functions or global variables from `main.cpp` into `appname.cpp`. Anything that was a global variable should be declared as static, this scopes them to the `.cpp` file. This is to avoid polluting the `Isis` namespace and avoid redefining symbols. 
```